hw02/hw02.R

```diff
@@ -0,0 +1,64 @@
+library(dplyr)
+library(gapminder)
+gapminder$country
+#1.1
+data1.1 <- gapminder %>% 
+  filter(country == "Afghanistan"|country == "Albania"|country == "Angola", year < 1980 & year > 1969)  
+#1.2
+data1.1 %>% 
+  select(country, gdpPercap)
+#1.3
+lifeExpchange1 <- gapminder %>% 
+  arrange(year) %>% 
```

Comment:
I added **group_by(country)** here so that life expectancy difference scores weren't being subtracted across countries